python/jetToolbox_cff.py

```diff
@@ -700,6 +713,10 @@ def jetToolbox( proc, jetType, jetSequence, outputFile,
 	elemToKeep += [ 'drop *_selectedPatJets'+jetALGO+'PF'+PUMethod+'_calo*_*' ]
 	elemToKeep += [ 'drop *_selectedPatJets'+jetALGO+'PF'+PUMethod+'_tagInfos_*' ]
 
+	if runOnData:
+		from PhysicsTools.PatAlgos.tools.coreTools import removeMCMatching
+		removeMCMatching(proc, names=['Jets'], outputModules=[])
```

Comment:
There's a bug in the ``removeMCMatching`` function. It crashes telling me that it can't find the output module named 'outputFile' (which is normal because it does not exists).

Since AFAIK the ``outputFile`` output module is not used anywhere, it should not matter to pass an empty array here, but I might be wrong. Do you see a case where we can have issues?